BatteryMeasure.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BatteryManagement
+{
+    class BatteryMeasure
+    {
+        public float Temperature, StateOfCharge, ChargeRate;
+        public BatteryMeasure(float temperature, float soc, float chargeRate)
+        {
+            this.Temperature = temperature;
+            this.StateOfCharge = soc;
+            this.ChargeRate = chargeRate;
+            BatteryLimitMessage.MeasureCrossedMaximum = new List<String>();
```

Comment:
Redundant Initialization of List .... check in BatteryLimitMessage

Reply:
Removed the Redundant Initialization of Lists in the BatteryLimitMessage file